### DISCUSSION TITLE
CSP errors - where is the CSP set?

### DISCUSSION THREAD

**Discussion by ghost** [apache/airflow#36823] [Q&A]
Hello,
I have a lot of CSP errors when I want to visit the grid view.
I am sure that I do not set any CSP because I get the errors when directly accessing the website on port 8080 

I get the errors in firefox and chrome but not in Brave, hmm. strange.

Any ideas what this might be?

**Comment by potiuk** [comment]
I think you should start with explaining what do you mean by CSP error and show some example of those errors you get. It's not clear what kind of errors you get.

**Comment by ghost** [comment]
The CSP = content security policy. I checked the headers and the html for the meta tag but I didn't find any and I still get those CSP errors. They start from the index page of the dags and the grid view of a dag is not loading because of them.

Here is a screenshot.
![ksnip_20240117-144828](https://github.com/apache/airflow/assets/79454572/2a07f958-dfac-4a4b-8de3-4326484b36b5)

Thank you for helping @potiuk

**Comment by potiuk** [comment]
It's likely your firewall or proxy that you have injecting those headers. Nothing to do with Airflow.
